tendrl/gluster_integration/message/gluster_native_message_handler.py

```diff
@@ -0,0 +1,97 @@
+from flask import Flask
+from flask import request
+import gevent
+import gevent.event
+import gevent.greenlet
+from gevent.wsgi import WSGIServer
+
+from tendrl.commons.utils import cmd_utils
+from tendrl.commons.utils import log_utils as logger
+from tendrl.commons.utils import service as svc
+from tendrl.gluster_integration.message import callback as cb
+
+
+app = Flask(__name__)
+
+
+class GlusterNativeMessageHandler(gevent.greenlet.Greenlet):
+    def __init__(self):
+        super(GlusterNativeMessageHandler, self).__init__()
+        self.path = "/listen"
+        self.host = "0.0.0.0"
+        self.port = 9000
```

Comment:
make this port 8697

9000 is too common

Reply:
ack